recipes/windows_service.rb

```diff
@@ -21,5 +21,6 @@
 return unless platform? 'windows'
 
 service 'WinRM' do
```

Comment:
can you ignore the foodcritic error on this resource please:
 `service 'WinRM' # ~FC009`

Reply:
Why don't just use `windows_service` resource?

Reply:
What do you think @Annih ?